versions/3.1.0.md

```diff
@@ -1535,10 +1536,9 @@ When passing in `multipart` types, boundaries MAY be used to separate sections o
 
 * If the property is a primitive, or an array of primitive values, the default Content-Type is `text/plain`
 * If the property is complex, or an array of complex values, the default Content-Type is `application/json`
-* If the property is a `type: string` with a `contentEncoding`, the default Content-Type is `application/octet-stream`
-* If the JSON Schema keyword `contentMediaType` is used and no Encoding Object is present, then the Content-Type is that which is specified by `contentMediaType`, however if an Encoding Object is present, then `contentMediaType` SHALL be ignored
+* If the property is a `type: string` with a `contentEncoding`, the default Content-Type is `text/plain`, and the media type of the embedded resource is specified in `contentMediaType`
```

Comment:
```suggestion
* If the property is a `type: string` with a `contentEncoding`, the default Content-Type is `application/octet-stream`
```
Changed this back to match wording here: https://tools.ietf.org/html/draft-bhutton-json-schema-validation-00#section-8.3